hs_core/hydroshare/utils.py

```diff
@@ -754,6 +774,20 @@ def resource_file_add_process(resource, files, user, extract_metadata=False,
     return resource_file_objects
 
 
+def create_empty_contents_directory(resource):
+    if resource:
```

Comment:
Yes, I think so. It's always nice to make sure the input object is not empty before doing anything.